lib/Transitions/ColorTransition.cpp

```diff
@@ -122,10 +121,9 @@ void ColorTransition::step() {
     callback(bulbId, GroupStateField::SATURATION, parsedColor.saturation);
     lastSaturation = parsedColor.saturation;
   }
-
-  if (currentColor == endColor) {
-    finished = true;
-  } else {
+  
+  if(!isFinished())
+  {
     Transition::stepValue(currentColor.r, endColor.r, stepSizes.r);
```

Comment:
```suggestion
  
  if (!isFinished()) {
    Transition::stepValue(currentColor.r, endColor.r, stepSizes.r);
```

Just to keep styling consistent :)